switchmap/poller/snmp/async_snmp_manager.py

```
@@ -0,0 +1,1100 @@
+    usmHMAC128SHA224AuthProtocol,
+    usmHMAC192SHA256AuthProtocol,
+    usmHMAC256SHA384AuthProtocol,
+    usmHMAC384SHA512AuthProtocol,
+    # Privacy protocols
+    usmDESPrivProtocol,
+    usmAesCfb128Protocol,
+    usmAesCfb192Protocol,
+    usmAesCfb256Protocol,
+)
+
+from pysnmp.proto.errind import RequestTimedOut
```

Ruff (0.12.2)

`pysnmp.proto.errind.RequestTimedOut` imported but unused

Remove unused import: `pysnmp.proto.errind.RequestTimedOut`

(F401)